interfaces/builtin/gconf.go

```diff
@@ -0,0 +1,74 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const gconfSummary = `allows access to any gconf item of the current user`
```

Comment:
this description should probably contain the word legacy somehow

Reply:
@jdstrand would it be unfair/unclear to turn that into `allows access to any legacy gconf item of the current user`? or otherwise `allows access to any item from the legacy gconf subsystem for the current user`?

Reply:
I used the same language that we have in the gsettings interface. Let me come up with something with legacy and push it up. (No, I don't think it is unfair, it is a legacy system).

Reply:
I landed on "allows access to any item from the legacy gconf configuration system for the current user". Happy to adjust as needed.